examples/blog/src/components/FollowMe.astro

```diff
@@ -0,0 +1,18 @@
+---
+export interface Props {
+	username: string;
+	href: string;
+}
+
+const { username, href } = Astro.props as Props;
+---
+
+<div class="author">
```

Comment:
(personal preference, feel free to ignore 😅 )

I like to avoid classes whenever possible and let scoped styles take care of it for me with a plain old `div { }` selector

Reply:
No problem! While we're at it, let's remove the `div` and just use `p` 👍